compression-webpack-plugin/compression-webpack-plugin-tests.ts

```diff
@@ -0,0 +1,14 @@
+import { Configuration } from 'webpack'
+import * as CompressionPlugin from 'compression-webpack-plugin'
```

Comment:
Please use the ```import CompressionPlugin = require('compression-webpack-plugin')``` syntax, as that's the one that works with ```export = somefunction``` type exports ([reference](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require))